module/checker.go

```diff
@@ -11,9 +11,20 @@ import (
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/duration"
 	"github.com/golang/protobuf/ptypes/timestamp"
-	"github.com/lyft/protoc-gen-star"
+	pgs "github.com/lyft/protoc-gen-star"
 )
 
+var unknown = ""
+var httpHeaderName = "^:?[0-9a-zA-Z!#$%&'*+-.^_|~\x60]+$"
+var httpHeaderValue = "^[^\u0000-\u0008\u000A-\u001F\u007F]*$"
```

Comment:
It looks like these regular expressions are duplicated here and in the Python code. Is it possible to unify those so we can have a single source of truth?

Reply:
The python code doesn't go through the go code checkers, but possibly I could have a const file with these regexes that python parses? Or is there a way that i can define constants in a protobuf file (`validate.proto`)?

Reply:
I think defining these in a separate file is heading towards the right solution, but instead of a .proto file it can be a textproto or YAML file that we check in. The contents would be of a new message type that is basically just `map<KnownRegex, string>` and contains the single canonical definition of the regex for each well-known regex. Then we can refer to that during Go/C++/Java code generation and include it in the Python code (not completely clear on how, but it should be possible).

Reply:
This might be too much for this PR, though. Up to you whether you want to do that here; if not, please open another issue to track cleaning up the tech debt.

Reply:
Oh, interesting. I like the textproto. It would make sense to include uuid patterns / other regexes that are duped (although the uuid dupe might be able to removed in code). 

I filed an issue https://github.com/envoyproxy/protoc-gen-validate/issues/316 and I'm happy to work on next week (after I use this PR in envoy!)